core/src/main/java/com/microsoft/applicationinsights/channel/concrete/inprocess/InProcessTelemetryChannel.java

```diff
@@ -61,11 +61,17 @@
  */
 public final class InProcessTelemetryChannel implements TelemetryChannel {
     private final static int DEFAULT_NUMBER_OF_TELEMETRIES_PER_CONTAINER = 500;
+    private final static int MIN_NUMBER_OF_TELEMETRIES_PER_CONTAINER = 1;
+    private final static int MAX_NUMBER_OF_TELEMETRIES_PER_CONTAINER = 500;
 
     private final static int TRANSMIT_BUFFER_DEFAULT_TIMEOUT_IN_SECONDS = 10;
+    private final static int MIN_TRANSMIT_BUFFER_TIMEOUT_IN_SECONDS = 1;
+    private final static int MAX_TRANSMIT_BUFFER_TIMEOUT_IN_SECONDS = 30;
```

Comment:
Regarding the max values if the user decides to override these then I'm not sure that we should limit him, and even if we do I think that these limits should be about 10 times higher, e.g. ~5000 telemetries per container and ~300 seconds per transmition